### PR TITLE
Kudu home page fixes

### PR DESCRIPTION
**Pull request by davidebbo**
Shaw runtime version + don't show uptime milliseconds

### CODE REVIEW

File: Kudu.Services.Web/Default.cshtml [hunk 1]
```diff
@@ -1,11 +1,31 @@
...
+        Assembly assembly;
+        try
+        {
+            assembly = Assembly.Load("Microsoft.Web.Hosting, Version=7.1.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35");
+            var fileVersionInfo = System.Diagnostics.FileVersionInfo.GetVersionInfo(assembly.Location);
+            return fileVersionInfo.ProductVersion;
+        }
+        catch
+        {
+            return null;
```
Comment (shrimpy) [inline Kudu.Services.Web/Default.cshtml]: shall we return empty string or a nicely error message instead of display null?

Comment (davidebbo) [inline Kudu.Services.Web/Default.cshtml]: It's not an error. It's the normal behavior on non Azure machines.

Comment (shrimpy) [inline Kudu.Services.Web/Default.cshtml]: if we return null. on the page, will it display "null" or empty? maybe should return empty string or "-" instead?

Comment (davidebbo) [inline Kudu.Services.Web/Default.cshtml]: See below, the whole section is omitted when it's null.

Comment (shrimpy) [inline Kudu.Services.Web/Default.cshtml]: ah. didn`t pay attention to "appServiceVersion" in line 11. thx

**Comment by shrimpy** [comment]
:shipit:

**Pull request merged**